### PR TITLE
add mathInteraction.pg to orientation

### PR DESCRIPTION
**Pull request by Alex-Jordan**
This adds a problem to the Student Orientation that discusses some things about MathJax. This was motivated by https://github.com/openwebwork/webwork2/discussions/2459.

If the display mode is `images` or `plainText`, the problem text basically suggests there is not much interaction with the math that is possible, and the user should look into using MathJax.

While updating the set def file for the orientation, I noticed trailing spaces. This turns out to come from a space in `lib/WeBWorK/File/SetDef.pm` that I think need not be there. So I removed that too.

While I was doing this, I fixed one typo in `assets/pg/Student_Orientation/hardcopy.pg` and made a minor wording edit in `assets/pg/Student_Orientation/enteringMath.pg`.

### CODE REVIEW

File: assets/pg/Student_Orientation/mathInteraction.pg [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+[!Speech string for the quadratic formula!]{'explorerfull.png'}{600}
+* Use the down arrow to navigate "down" into a smaller piece of the math expression.
+* Use the left/right arrows to navigate to similar pieces of the math expression.
+* At any time you can navigate back "up" to a larger part of the expression, or "down" into smaller pieces. For example,
+you can see a verbalization for just this part of the expression above:  
+[!Speech string for the radicand of the quadratic formula!]{'explorerpiece.png'}{600} 
+* Return to the MathJax menu, Accessibility submenu, to explore options for how this explorer tool works.
+* Under "Speech" you will find options to use MathSpeak, ClearSpeak, or ChromeVox rules. The default is to use
+"MathSpeak verbose" rules, which try try to read math "literally" without context. For example, it reads [`(1,3)`] as
+"left parenthesis 1 comma right parenthesis". Other speech rules can produce more meaningful verbal renderings. For
```
Comment (taniwallach) [inline assets/pg/Student_Orientation/mathInteraction.pg:52]: Should it be "1 comma 3" in line 52?

**Review by taniwallach — COMMENTED** [review]
I like the idea of providing the explanation about MathJax accessibility. etc. for people who need it.

File: assets/pg/Student_Orientation/mathInteraction.pg (hunk 1 above)
Comment (Alex-Jordan) [reply assets/pg/Student_Orientation/mathInteraction.pg]: Thanks for seeing that. I updated.

File: assets/pg/Student_Orientation/mathInteraction.pg [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+DOCUMENT();
+
+loadMacros(qw(
+    PGstandard.pl
+    PGML.pl
+    parserRadioButtons.pl
+    PGcourse.pl
+));
+
+$continue = RadioButtons(['I am ready to continue. Sumbmit this answer.'], 0);
```
Comment (drgrice1) [inline assets/pg/Student_Orientation/mathInteraction.pg:14]: Typo: Sumbmit -> Submit

**Review by drgrice1 — APPROVED** [review]
Looks good.

**Review by somiaj — APPROVED** [review]
Looks good, just a typo to fix.

File: assets/pg/Student_Orientation/mathInteraction.pg (hunk 2 above)
Comment (Alex-Jordan) [reply assets/pg/Student_Orientation/mathInteraction.pg]: OK, should be fixed now.

**Pull request merged**